.github/workflows/format.yml

```diff
@@ -30,6 +30,6 @@ jobs:
                   fetch-depth: 0
                   ref: ${{ github.event.pull_request.head.sha }}
             - name: Install clang-format
-              run: sudo ./.github/scripts/install-clang-format.sh 14
+              run: sudo ./.github/scripts/install-clang-format.sh 15
```

Comment:
This is really inconvenient for me as clang-format 15 is not currently available on homebrew :/ However, this is not your fault and homebrew _should_ offer the current version. I will see what I can do...

Reply:
Yes sry, insert braces seems to be available only from 15 onwards.

Reply:
A [patch for homebrew](https://github.com/Homebrew/homebrew-core/pull/114281) is on the way, and for the meantime, I have compiled a version of clang-format 15 locally :-)